inference-engine/src/readers/ir_reader/ie_ir_parser.hpp

```diff
@@ -251,6 +256,36 @@ class V10Parser : public IParser {
             stringToType<double>(val, value);
             adapter.set(value);
         }
+        void on_adapter(const std::string& name, ngraph::ValueAccessor<void*>& adapter) override  {
```

Comment:
Am I right that it is an adapter for void* array? On other words it used only to read weights.

Reply:
Yes, it is an adapter for void*.